avro/README.md

```diff
@@ -111,6 +111,36 @@ byte[] avroData = mapper.writer(schema)
 
 and that's about it, for now.
 
+## Java Time Support
+Serialization and deserialization support for limited set of `java.time` classes to Avro with [logical type](http://avro.apache.org/docs/current/spec.html#Logical+Types) is provided by `AvroJavaTimeModule`.
+
+```java
+AvroMapper mapper = AvroMapper.builder()
+    .addModules(new AvroJavaTimeModule())
```

Comment:
`addModule()` (I can remove `s`, just noting)

Reply:
Changed to `addModule`.